test/e2e/kuttl/ScaleUpAndDownDC/00-createCluster.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: helm install casskop orange-incubator/cassandra-operator
+    namespaced: true
+---
+apiVersion: db.orange.com/v1alpha1
+kind: CassandraCluster
+metadata:
+  name: cassandra-e2e
+spec:
+  nodesPerRacks: 1
+  cassandraImage: cassandra:latest
+  dataCapacity: "1Gi"
+  hardAntiAffinity: false
+  deletePVC: false
+  autoPilot: true
+  autoUpdateSeedList: false
+  bootstrapImage: orangeopensource/cassandra-bootstrap:0.1.7
```

Comment:
why do you need to specify it for the tests ?

Reply:
I ran into a version asssertion probleme with this image during my local tests (nothing asked and version 0.1.5 was used wich created assertion error). I think we can delete it.

Reply:
Option deleted